### PR TITLE
Changelog for 10.0.1

### PR DESCRIPTION
**Pull request by PVince81**
@lefherz @jnweiger to deploy around the announcement for 10.0.1

### CODE REVIEW

**Comment by felixboehm** [comment]
Please add link to release notes, which informs about breaking changes important for updates

**Comment by PVince81** [comment]
Added here https://github.com/owncloud/owncloud.org/pull/1218/commits/a87d18edf669c848b3faf93b1ea7aa7d32ea6399

Assuming the matching page will appear soon

**Comment by lefherz** [comment]
sorry, didn't see your comments. was this too early? @PVince81

**Comment by PVince81** [comment]
Merging is ok. It doesn't put it live.

Deploying requires another step which can be done later.

cc @jnweiger

**Comment by lefherz** [comment]
right, that's why i thought merging was okay :) lets review it on staging.

File: page-changelog.php [hunk 1]
```diff
@@ -73,6 +73,7 @@
 <li>[minor] Improve visual feedback when installing market apps - <a href="https://github.com/owncloud/market/issues/64">market/#64</a></li>
 <li>[minor] Don't display license key in config report - <a href="https://github.com/owncloud/configreport/issues/27">configreport/#27</a></li>
 </ul>
+Release notes: <a href="https://doc.owncloud.org/server/10.0/admin_manual/release_notes.html#changes-in-10-0-1">Release notes</a></br>
```
Comment (butonic) [inline page-changelog.php]: Can you make the release notes more prominent and add something like 'The internal unermanagement has changed fundamentally, please read the release notes!'

Comment (PVince81) [inline page-changelog.php]: We could move it up but I don't have a local wordpress deploy to find out whether it looks good or breaks the layout.

Do one of you guys have it ? @stkjj @lefherz 
Feel free to move it

Comment (lefherz) [inline page-changelog.php]: Is it here? what do you need a wordpress account for? https://staging.owncloud.org/changelog/#latest10.0
it looks fine here, at least :) link to the release notes worked.

Comment (PVince81) [inline page-changelog.php]: Ok, so move the link up under the title and before the list items ? @lefherz

Comment (lefherz) [inline page-changelog.php]: commited it directly, I failed to create a new branch for a PR^^ does it look fine on staging?

**Pull request merged**